Admin/CategoryAdmin.php

```diff
@@ -18,10 +18,6 @@
 
 class CategoryAdmin extends ContextAwareAdmin
 {
-    protected $formOptions = array(
```

Comment:
More details about this in [the commit message](https://github.com/sonata-project/SonataClassificationBundle/pull/250/commits/1d98b7549b420bf53faabd1871cde38aed26e47e)